versioned_docs/version-2.6/reference-guides/cluster-configuration/rancher-server-configuration/rke1-cluster-configuration.md

```diff
@@ -119,15 +119,15 @@ Rancher v2.6 introduced the ability to configure [ECR registries for RKE cluster
 
 ### Authorized Cluster Endpoint
 
-Authorized Cluster Endpoint can be used to directly access the Kubernetes API server, without requiring communication through Rancher.
+Authorized Cluster Endpoint (ACE) can be used to directly access the Kubernetes API server, without requiring communication through Rancher.
 
 :::note
 
-The authorized cluster endpoint only works on Rancher-launched Kubernetes clusters. In other words, it only works in clusters where Rancher [used RKE](../../../reference-guides/rancher-manager-architecture/communicating-with-downstream-user-clusters.md#tools-for-provisioning-kubernetes-clusters) to provision the cluster. It is not available for clusters in a hosted Kubernetes provider, such as Amazon's EKS.
+ACE is only available on Kubernetes clusters launched by Rancher. In Rancher v2.6.3 and later, this means clusters where Rancher used RKE, RKE2, or K3s to provision the cluster. Prior to Rancher v2.6.3, it was only available for RKE. It's also not available for clusters in a hosted Kubernetes provider, such as Amazon's EKS.
 
 :::
 
-This is enabled by default in Rancher-launched Kubernetes clusters, using the IP of the node with the `controlplane` role and the default Kubernetes self signed certificates.
+ACE must be set up [manually](../../../how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/register-existing-clusters#authorized-cluster-endpoint-support-for-rke2-and-k3s-clusters.md) on RKE2 and K3s clusters in Rancher v2.6.3 and later. In RKE, ACE is enabled by default in Rancher-launched Kubernetes clusters, using the IP of the node with the `controlplane` role and the default Kubernetes self signed certificates.
```

Comment:
```suggestion
ACE must be set up [manually](../../../how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/register-existing-clusters#authorized-cluster-endpoint-support-for-rke2-and-k3s-clusters.md) on RKE2 and K3s clusters in Rancher v2.6.3 and later. In RKE, ACE is enabled by default in Rancher-launched Kubernetes clusters, using the IP of the node with the `controlplane` role and the default Kubernetes self-signed certificates.
```

Reply:
Hmm, we link here to the RKE2/K3s docs for registering existing clusters, and talk about enabling ACE on these clusters. It sounds like some of the previous language about only supporting ACE on clusters provisioned is not correct, and it can in fact be enabled on an RKE2 or K3s cluster. Can we verify that?

Reply:
@aiyengar2 Can you or someone else on the team confirm the above comment?

Reply:
I did a quick check on this and saw that we can register (import) existing RKE2/k3s clusters and was able to enable/disable ACE on both of these cluster types.

Reply:
From Slack:

> AFAIK it still will not work for hosted providers. We have special integrations for k3s/rke2 as those are our distros